src/styles/pages/about.scss

```diff
@@ -1,38 +1,218 @@
 @import '../common';
 
 .PingCAP-About {
+  $is-12: 80vw;
```

Comment:
How did you calculate this value? It seems not correct

Reply:
80vw = the width of 12 columns(84rem) / the width of page width(105rem), I get them from design draft

Reply:
> How did you calculate this value? It seems not correct

80vw = the width of 12 columns(84rem) / the width of page width(105rem), I get them from design draft

Reply:
since the width of page is different from desktops (13 inch, 15 inch, monitor display and so on), you cannot set the width of 12 columns to 80vw. 

the width of 12 columns are the max-width of each [breakpoints](https://bulma.io/documentation/layout/container/) . So you don't need to care the width of 12 columns

Reply:
> since the width of page is different from desktops (13 inch, 15 inch, monitor display and so on), you cannot set the width of 12 columns to 80vw.
> 
> the width of 12 columns are the max-width of each [breakpoints](https://bulma.io/documentation/layout/container/) . So you don't need to care the width of 12 columns

ok, i get it, i'll fix it later